lib/generators/shopify_app/app_proxy_controller/templates/app_proxy_controller.rb

```diff
@@ -0,0 +1,12 @@
+class AppProxyController < ApplicationController
+  if Rails.env.development? || Rails.env.test?
+   around_action :shopify_session
+  else
+   include ShopifyApp::AppProxyVerification
+  end
```

Comment:
The reason for this move was multi-fold:
- Since there's no way to access currently active shop's theme via localhost, there's actually no point in sending a special content-type header. 
- I dont have a Liquid Template/Parse-Render Engine defined for my local server. So during development, everytime I tried accessing the page with Content-type: 'application/liquid', the page would automatically get downloaded as a file, by the browser, instead.
- I assumed that the new app developers trying their hand at App Proxy may not have their local server to handle liquid headers either and hence would instead show them a default render.

Yes, I checked production status via a dev shop.